.github/workflows/python-env-upload.yml

```diff
@@ -0,0 +1,44 @@
+# This workflows will upload a python environment container to quay when there are changes to master
+# For more information see: https://github.com/docker/build-push-action
+
+name: Upload Python Environment Container to Quay
+
+on:
+  push:
+    branches: master
+  pull_request:
+
+jobs:
+  multi:
+    runs-on: ubuntu-latest
+    steps:
+      - 
+        name: Get smart tag
+        id: prepare
+        uses: Surgo/docker-smart-tag-action@v1
+        with:
+          docker_image: quay.io/multicloudops/operator-csv-libs
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - 
+        name: login to quay
+        run: echo "${{ secrets.QUAY_TOKEN }}" | docker login quay.io -u "${{ secrets.QUAY_USER }}" --password-stdin
```

Comment:
It looks like this step should also be conditional so that it doesn't run on pull request, since it appears secrets are not available for them (nor is a login required)
https://github.com/multicloud-ops/operator-csv-libs/pull/10/checks?check_run_id=1231962384